vault/namespaces_store.go

```diff
@@ -35,6 +35,9 @@ const (
 	// Namespace storage location.
 	namespaceStoreRoot = "core/namespaces/"
 
+	// Namespaces lock location.
+	lockPrefix = "core/namespace-lock"
```

Comment:
```suggestion
	// Namespaces lock location.
	namespaceLockPrefix = "core/namespace-lock"
```

since this is a global symbol within the `vault` package.